reference/hoon-expressions/advanced.md

```diff
@@ -188,13 +191,13 @@ of secret superpowers for hacking the namespace.  Remember that
 Hoon doesn't have anything like a symbol table; to resolve a
 limb, we just search the type depth-first.
 
-If a name is in the `p.p` map, it's an alias.  (An alias is defined using the
+If a name is in the `p.q.p` `map`, it's an alias.  (An alias is defined using the
 `=*` rune.) The map contains a `(unit hoon)`; if the unit is full, the name
 resolves to that hoon (compiled against the `q` type).  If the unit is empty,
 the name is blocked / skipped (see [limb](@/docs/reference/hoon-expressions/limb/limb.md) for what
 this means).
 
-If a name is in the `q.p` map, it's a bridge.  (A bridge is defined using the
```

Comment:
Here `q.p` seems correct per my above comment, but as you point out, the result is not a map but a `(list hoon)`.  Perhaps `$tune` used to be implemented like this.

What happens now seems to be similar in spirit to what is described here, but the details are slightly different and more involved, and are probably unimportant for intuition.  See the `%tscm` case in `+play` and note the call to `+busk`.